### ISSUE TITLE
geoclue2 failing to find /etc/geoclue/conf.d

### ISSUE THREAD

**Issue by fzakaria** [NixOS/nixpkgs#327464]
### Describe the bug
My geoclue2 service was emitting the following message:
```
Jul 15 19:25:18 nyx .geoclue-wrappe[2170]: Failed to open /etc/geoclue/conf.d/: Error opening directory “/etc/geoclue/conf.d/”: No such file or directory
```

I got around it with

```
  environment.etc."geoclue/conf.d/.valid" = {
    text = "created the directory.";
  };
```

### Expected behavior
The service shouldn't be emitting such messages

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by rvl** [comment]
I opened an [upstream issue](https://gitlab.freedesktop.org/geoclue/geoclue/-/issues/198) because I think this is more a bug in Geoclue than NixOS.

**Comment by stephen-huan** [comment]
It looks like it has already been fixed and will probably appear in the next release.

https://gitlab.freedesktop.org/geoclue/geoclue/-/merge_requests/169

Since this issue was mostly cosmetic in the first place, I'll remove the fix from the PR.

**Comment by Axolord** [comment]
Can we back port that fix from https://github.com/NixOS/nixpkgs/pull/352480 to 24.05?

**Comment by zane-weissman** [comment]
> Can we back port that fix from #352480 to 24.05?

Not sure what the backporting policy is for nixpkgs, but in the meantime you can easily [include a package from the unstable channel.](https://discourse.nixos.org/t/2-channels-in-1-system-is-it-possible/10679/2)
